common/src/config.rs

```diff
@@ -307,6 +307,8 @@ struct Cache {
     batch_interval: u32,
     /// Interval for watching local cache override files in seconds.
     file_interval: u32,
+    /// Jitter to be applied to cache timeouts (such as projects).
```

Comment:
Can we document the valid range of this? I.e. `0 - 1`, not `-1 - 1` or something else

Reply:
Done.